packages/eslint-plugin-clarity-adoption/src/rules/no-clr-alert/index.ts

```diff
@@ -12,7 +12,6 @@ export default createESLintRule({
     type: 'problem',
     docs: {
       description: 'Disallow use of clr-alert',
-      category: 'Best Practices',
```

Comment:
`meta.docs.category` has been removed from eslint the typescript-eslint package. See more [here](https://github.com/typescript-eslint/typescript-eslint/releases/tag/v5.0.0).